ddtrace/tracer/tracer.go

```diff
@@ -664,6 +671,14 @@ func (t *tracer) StartSpan(operationName string, options ...ddtrace.StartSpanOpt
 			log.Error("Abandoned spans channel full, disregarding span.")
 		}
 	}
+	t.spansStarted.mu.Lock()
+	defer t.spansStarted.mu.Unlock()
+	if t.spansStarted.spans == nil {
+		t.spansStarted.spans = make(map[string]uint32)
+	}
+	count := t.spansStarted.spans[span.integration]
+	atomic.AddUint32(&count, 1)
```

Comment:
I don't think you need to use atomic here, since you've already managed a potential race condition with the lock.